stress/shell/src/main/java/alluxio/stress/cli/Benchmark.java

```diff
@@ -230,6 +233,46 @@ public String run(String[] args) throws Exception {
     return nameStatistics;
   }
 
+  protected static final class JavaAgentInput {
+    private String mType;
```

Comment:
these all should be `final`.

Reply:
do you want me to remove the `setMethod()`?

Reply:
changed.